static/app/views/issueDetails/groupReplays/groupReplays.tsx

```diff
@@ -27,14 +29,40 @@ const VISIBLE_COLUMNS = [
   ReplayColumn.ACTIVITY,
 ];
 
-function GroupReplays({group}: Props) {
+function GroupReplays({group, event}: Props) {
+  const now = useMemo(() => Date.now(), []);
```

Comment:
this should be a useRef since it isn't updated or used to rerender. also consider storing the iso string 
```suggestion
  const now = useRef(new Date().toISOString())
```